ci/packet/packet-cluster.lokocfg.envsubst

```diff
@@ -22,6 +22,7 @@ cluster "packet" {
   node_private_cidr = "10.0.0.0/8"
 
   worker_pool "pool-1" {
+    ssh_pubkeys       = ["$PUB_KEY"]
```

Comment:
Commenting here as I can't comment on the commit itself (github limitation): Why split the commit with one that has "all changes, including x86 in the CI, except ARM on the CI" and then "Fix ARM on the CI"?

Reply:
removed this commit as its not needed. `ssh_pubkeys` is optional and gets set to the one provided for the controller if not specified for workerpool.